prime-router/metadata/hl7_mapping/ORU_R01/base/datatype/xon-organization.yml

```diff
@@ -3,19 +3,19 @@ elements:
     value: [ '%resource.name' ]
     hl7Spec: [ '%{hl7OrgField}-1' ]
 
-  - name: xon-organization-type
+  - name: xon-organization-name-type
```

Comment:
Looks like there is one more dupe.  Look for set-id in patient.  Thanks for cleaning all of these.

Reply:
Fixed all missing dupes